### ISSUE TITLE
Bug when dumping particles to openpmd format

### ISSUE THREAD

**Issue by roussel-ryan** [desy-ml/cheetah#510]
Recieving the a zero length error when aiming to dump data to file https://github.com/desy-ml/cheetah/blob/01f4aed7f8f6733eb48accb057be19f4904f9d47/cheetah/particles/particle_beam.py#L919

Need to wrap `particle_charges` with numpy atleast1d function

**Comment by roussel-ryan** [comment]
Might also want to add a `.detach()` and `.cpu()` before converting to numpy
